<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="TitleLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Diogenes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="InstanceTypeComboBox">
      <property name="currentText">
       <string/>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="VolumeSizeLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="DeleteStorageButton">
      <property name="text">
       <string>Delete Storage On Termination </string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RequestInstanceButton">
      <property name="text">
       <string>Launch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="InfoLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quick Links: &lt;br/&gt;• &lt;a href=&quot;http://diogenes-redirect-ec2-spot-pricing.s3-website-us-east-1.amazonaws.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;EC2 Spot Pricing&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://aws.amazon.com/ec2/spot/pricing/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;• &lt;a href=&quot;http://diogenes-redirect-ec2-pricing.s3-website-us-east-1.amazonaws.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;EC2 On-Demand Pricing&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://aws.amazon.com/ec2/pricing/on-demand/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;• &lt;a href=&quot;http://diogenes-redirect-ec2-instance-types.s3-website-us-east-1.amazonaws.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;EC2 Instance Types&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://aws.amazon.com/ec2/instance-types/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;• &lt;a href=&quot;https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-attaching-volume.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;Attach Existing EBS Volume&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
